doc/source/metadata/Introduction.rst

```diff
@@ -0,0 +1,724 @@
+Introduction
+============
+
+.. contents:: Contents
+    :depth: 3
+
+Overview
+--------
+
+Metadata is data that describes other data.
+
+In IMPROVER this takes the form of attributes either within a netCDF file
+or within an in-memory Iris cube. 
+
+The principles applied to the metadata witin IMPROVER are: 
+
+ * Conformance to the CF Metadata Conventions, building on this where necessary
+ * Clear purpose, with just enough metadata to describe the data sufficiently
+ * Nothing misleading or unnecessarily restrictive for wide usage
+ * (Ideally) support for referencing more detailed external documentation
+
+Looking at an example
+---------------------
+
+The easiest way to explain the IMPROVER metadata from a user perspective
+is to dive straight in and look at an actual example of the metadata.
+For this purpose, we will consider
+gridded probabilities for a 12-hour maximum temperature 
+exceeding a range of thresholds
+on an extended UK domain generated from Met Office model data. 
+
+There are two common views of the metadata:
+
+* How the ncdump utility would display the netCDF file metadata 
+* How iris would display the cube
+
+We will mainly focus on the file view as here as the default output
+provides a fuller view
+(although iris can be used to fully explore the metadata).
+
+Note that to aid readability, in both cases
+``probability_of_air_temperature_above_threshold`` 
+has been shortened to ``prob_temp12m``
+and for the ncdump output trailing semicolons have also been removed
```

Comment:
This was mainly done to fit this on a page - originally this was in a Word document. I maintained it to avoid scroll bars on the code snippets, but agree that a more pure approach is to show people what they would actually see when invoking an ncdump command, so I'll revert these to pure copies of the ncdump files (including semicolons, full titles and without artificial formatting or line breaks).